data/repos.yml

```diff
@@ -402,6 +514,9 @@
   type: Utilities
   retired: true
 
+# Propose archiving - prototype, not touched since July 2020
+- repo_name: govuk-datalabs-eligibility-prototype
```

Comment:
@nacnudus

Reply:
Should be archived, but I don't have the power.

Reply:
Thanks, I've [asked in #govuk-tech-leads](https://gds.slack.com/archives/CD6JUFXML/p1679902015818569) if anyone can help.